### PR TITLE
Maximum ETH deposits/withdrawals ignore the gas fee

### LINKED ISSUE #12: Maximum ETH in UX ignores gas fee
### Problem description
When a user fills in the maximum allowed amount in the amount field in the bridge view, the deposit cannot be executed due to there not being any ETH left for gas. It's only possible to reject the transaction and start over with a lower amount.

### Expected behaviour
Catch this issue earlier by taking into account gas pricing. The amount a user deposits is the deposit + gas fee.

### PR DESCRIPTION
**Pull request by HDauven**
**Purpose**
Fix a situation where the maximum ETH in the UX ignores the gas fee, resulting in there being no funds left to bridge. This solution checks the balances and gas costs pre-transaction execution for low balance wallets, resulting in simplified implementations for #25 and #26.

**Changes**
- Maximum ETH deposits now take a safe gas cost limit into account.
- Max button has been added to max out deposits or withdrawals.
- Reset UI values for certain edge case conditions.

**Issues**
Closes #12 #25 

**Check list**
- [x] Code runs without regressions even though new code is incomplete
- [x] Code reviewer has been explicitly notified outside automatic notification channels

### CODE REVIEW

File: src/lib/bridge/Bridge.svelte [hunk 1]
```diff
@@ -74,6 +75,7 @@
     $: deposit = L2 === false ? true : false;
 
     const selectToken = () => {
+        amountToBridge = "0";
```
Comment (MariusTH) [inline src/lib/bridge/Bridge.svelte:78]: You import the constant ZERO which is "0" so maybe use that one instead of "0"?

**Review by MariusTH — CHANGES_REQUESTED** [review]
Code looks good! Just nitpick on using "0" instead of the constant that is imported!

File: src/lib/bridge/Bridge.svelte (hunk 1 above)
Comment (HDauven) [reply src/lib/bridge/Bridge.svelte]: Done for all occurrences!

**Review by MariusTH — APPROVED** [review]
looks good!

**Pull request merged**